### ISSUE TITLE
Can't enable auto sync due to missing network settings

### ISSUE THREAD

**Issue by bassmannate** [koreader/koreader#11208]
* KOReader version: 2023.10
* Device: Linux

#### Issue
When trying to enable automatic progress sync, it tells me I have to switch on the 'Action when Wi-Fi is off' network setting to turn this on. When you go to network settings, clicking on it doesn't bring up anything. I don't have the same network settings as I do on my Android devices.
![image](https://github.com/koreader/koreader/assets/16216783/17286df8-d9da-4142-b3a1-c00b45f5f801)
![image](https://github.com/koreader/koreader/assets/16216783/b3cce353-54f8-4c56-8080-6a40c966de3f)

#### Steps to reproduce
Try to enable "automatically keep documents in sync' setting.

##### `crash.log` (if applicable)
N/A - No crash

`crash.log` is a file that is automatically created when KOReader crashes. It can normally be found in the KOReader directory:

* `/mnt/private/koreader` for Cervantes
* `koreader/` directory for Kindle
* `.adds/koreader/` directory for Kobo
* `applications/koreader/` directory for Pocketbook

Android logs are kept in memory. Please go to [Menu] → Help → Bug Report to save these logs to a file.

Please try to include the relevant sections in your issue description.
You can upload the whole `crash.log` file (zipped if necessary) on GitHub by dragging and dropping it onto this textbox.

If your issue doesn't directly concern a Lua crash, we'll quite likely need you to reproduce the issue with *verbose* debug logging enabled before providing the logs to us.
To do so, from the file manager, go to [Tools] → More tools → Developer options, and tick both `Enable debug logging` and `Enable verbose debug logging`.
You'll need to restart KOReader after toggling these on.

If you instead opt to inline it, please do so behind a spoiler tag:
<details>
  <summary>crash.log</summary>

```
<Paste crash.log content here>
```
</details>

**Comment by Frenzie** [comment]
Could you try the latest nightly? I think that was already fixed http://build.koreader.rocks/download/nightly/v2023.10-44-g3c63a5d8a_2023-12-04/

**Comment by bassmannate** [comment]
> Could you try the latest nightly? I think that was already fixed http://build.koreader.rocks/download/nightly/v2023.10-44-g3c63a5d8a_2023-12-04/

Yup, that did it. Thanks! Marking as resolved in nightly. Should hit the next release.
